src/main/java/com/palantir/gradle/revapi/OldApiConfigurations.java

```diff
@@ -18,16 +18,17 @@
 
 import com.palantir.gradle.revapi.config.GroupNameVersion;
 import com.palantir.gradle.revapi.config.Version;
-import java.io.File;
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
 import org.gradle.api.Project;
 import org.gradle.api.artifacts.Configuration;
 import org.gradle.api.artifacts.Dependency;
 import org.gradle.api.artifacts.result.DependencyResult;
 import org.gradle.api.artifacts.result.UnresolvedDependencyResult;
 
+import java.io.File;
```

Comment:
can you run `./gradlew spotlessApply` locally? This shouldn't be re-organizing the imports

Reply:
Yes I will.